wfe2/wfe.go

```diff
@@ -515,12 +515,17 @@ func (wfe *WebFrontEndImpl) Nonce(
 	}
 
 	statusCode := http.StatusNoContent
-	// The ACME specification says GET requets should receive http.StatusNoContent
+	// The ACME specification says GET requests should receive http.StatusNoContent
 	// and HEAD/POST-as-GET requests should receive http.StatusOK.
 	if request.Method != "GET" {
 		statusCode = http.StatusOK
 	}
 	response.WriteHeader(statusCode)
+
+	// The ACME specification says the server MUST include a Cache-Control header
+	// field with the "no-store" directive in responses for the newNonce resource,
+	// in order to prevent caching of this resource.
+	response.Header().Set("Cache-Control", "private, max-age=0, no-store")
```

Comment:
I don't _think_ we want private here? It's generally used to indicate the browser cache may store the value, which I can't see us wanting.

Similarly with `no-store` I'm not sure there is any reason to include `max-age=0`, using both is typically called out as redundant.

Reply:
Basically just `no-store` should be sufficient.

Reply:
Thanks Roland, I agree - `private` is *more* open than nothing, and `max-age` is redundant. I've removed both.